src/buildHTML.js

```diff
@@ -149,12 +149,8 @@ export const buildExpression = function(expression, options, isRealGroup,
     // Process \\not commands within the group.
     for (let i = 0; i < groups.length; i++) {
         if (groups[i].value === "\u0338") {
+            // Results in a solidus being overlaid over the following group/atom.
             groups[i].style.position = "absolute";
```

Comment:
Back to this approach, I wonder if this could be done when building the symbol, instead of a `for` loop after the fact?